freshmaker/pyxis.py

```diff
@@ -273,15 +273,16 @@ def get_images_by_digest(self, digest):
                           'filter': q_filter}
         return self._pagination('images', request_params)
 
-    def get_images_by_nvr(self, nvr):
+    def get_images_by_nvr(self, nvr, include_fields=[]):
         """
         Get images by image's NVR
 
         :param str nvr: NVR of image
+        :param list include_fields: included fields in image data.
         :return: images
         :rtype: list
         """
-        request_params = {"include": "data.architecture,data.brew,data.repositories"}
+        request_params = {'include': ','.join(include_fields)} if include_fields else {}
```

Comment:
The default `include` has been changed, it breaks the existing code, you can do it in this way:
```
request_params = {'include': 'data.architecture,data.brew,data.repositories'}
if include_fields:
    request_params.update({'include': ','.join(include_fields})
```

Reply:
Yeah. I am thinking to make the method more generic, so refactor it a bit. With a default request_params sounds ok for me.